package.json

```diff
@@ -297,5 +297,8 @@
 	},
 	"workspaces": [
 		"packages/*"
-	]
+	],
+	"dependencies": {
+		"eslint": "8.3.0"
+	}
```

Comment:
There should be no changes for this PR's `package.json` and `package-lock.json` files.

Please refer to the documentation for setting up code editor extensions.